src/main/java/com/aws/greengrass/deployment/DefaultDeploymentTask.java

```diff
@@ -196,7 +197,7 @@ public DeploymentResult call() throws InterruptedException {
         try {
             groupsForDeviceOpt = thingGroupHelper.listThingGroupsForDevice(retryCount);
         } catch (GreengrassV2DataException e) {
-            if (e.statusCode() == 403) {
+            if (e.statusCode() == HttpStatusCode.FORBIDDEN) {
```

Comment:
We can probably just import from aws sdk
 https://sdk.amazonaws.com/java/api/latest/software/amazon/awssdk/http/HttpStatusCode.html#FORBIDDEN